client/src/components/TeacherTable.js

```diff
@@ -1,17 +1,16 @@
 import React from "react"
 import StudentList from "./StudentList"
 
-function TeacherTable (){
-    return(
-       <div className="card">
-           <h1>Ms.Frizzle's Class--teacher's ID from DB here</h1>
-           <div className="row">
-               <ul className="list-group-item m-2">
-                  <StudentList />
-               </ul>  
-
-           </div>
-       </div>
+function TeacherTable(props) {
+    return (
+        <div className="card">
+            <h1>Ms.Frizzle's Class--teacher's ID from DB here</h1>
+            <div className="row">
```

Comment:
Is it necessary to have a row class if we're not using columns within? Might be extra code that doesn't do anything. May have affected it's children elements when we were trying to adjust/align the student name and attendance buttons. Just and FYI and something to consider with the next update.